Added sample Visualization for Radix Sort

rehassachdeva opened the pull request:
Please review. And suggest any modifications we should make.

CuriousLearner commented:
@rehassachdeva Okay Nice work there! 

But, while cloning it for testing, I was wondering why is it taking so long. So I noticed a few things:
- Why did you commit your virtual-env in your code?

It always goes in `.gitignore` file and you should never commit `virualenv`. What you do instead is create a `requirements.txt` file using `pip freeze`
- Why `contact_me.php` ? 

I wonder this is the same bootstrap files for `agency` theme. If you're not using/ probably not use them, why not remove them?
- Visualization looks good. You need to improve on them. Some people might feel it is a little bit faster? Some feel it is slow. So, try to add controllers.

Also, a few suggestions:

You could display the code somewhere on the screen, and if user wants to go line by line, then for each line you display how the animation works.

But your work is good so far. You can implement the suggestion part later. But first please remove the stuff that is not necessary and add `requirements.txt` file.

CuriousLearner commented:
@rehassachdeva Hadn't tried to look at your code before. I can see a lot of code copy pasted! While it is good when you're starting up to code, please try to understand what your code actually is doing.

Why are you returning posts and user at flask root? and the name miguel gave me idea where you might copied it from. 

PS: It's a very good resource to learn flask. But don't just add something which is not needed :)

rehassachdeva commented:
@CuriousLearner, very sorry about all that. I forgot to edit it. I'll remove the unnecessary files and code rightaway. Visualization code is completely original though. And thank you for the suggestions. will work on them in a while.

rehassachdeva commented:
@CuriousLearner Please check now

CuriousLearner commented:
@rehassachdeva Please remove your virtualenv from git commit and add a `requirements.txt` file so that anyone can make exact virtualenv on their machine.

You can do it via `pip freeze > requirements.txt` at root of the repo. You should be inside virtualenv while doing this.

Also, please mark the line endings and it is good to go!

CuriousLearner commented on coders4ever/app/__init__.py:
```diff
@@ -0,0 +1,4 @@
+from flask import Flask
+
+app = Flask(__name__)
+from app import views
```
Add a newline here

CuriousLearner commented on coders4ever/app/views.py:
```diff
@@ -0,0 +1,7 @@
+@app.route('/')
+@app.route('/index')
+def index():
+    return render_template("index.html")
```
Add a newline here too

rehassachdeva commented:
@CuriousLearner Yes I have added requirements.txt like that at the root, in a previous commit. And have removed newlines now and deleted virtualenv code. 

Thanks

CuriousLearner commented on coders4ever/run.py:
```diff
@@ -0,0 +1,3 @@
+#!flask/bin/python
+from app import app
+app.run(debug=True)
```
No newline here, please add.

CuriousLearner commented:
Also remove `*.pyc` files. Add an entry in `.gitignore` There is no need to commit pyc files :)

rehassachdeva commented:
@CuriousLearner done :)

CuriousLearner commented:
Okay, I'm merging this!

You can start implementing controllers if you want and make another PR.

Thanks for your work :+1:

rehassachdeva commented:
Thanks!

Yes we'll be working on controllers, displaying code snippets, ability to generate your own array, iteration count (or current radix used in the current iteration) and other enhancements, as soon as we are done with the application.

CuriousLearner commented:
Nice, all the best!
@rehassachdeva
